updater/types/types.go

```diff
@@ -31,8 +37,8 @@ var SystemApps = []SystemApp{
 	{Name: "controller"},
 	{Name: "logaggregator"},
 	{
-		Name:  "postgres",
-		Image: "flynn/postgresql",
+		Name:      "postgres",
+		ImageName: "postgresql",
```

Comment:
The image name is derived from the directory, so happy to move the code to `appliance/postgres`? Or we can just have an explicit `build-image --name postgres`.